integrated_channels/api/v1/moodle/serializers.py

```diff
@@ -12,8 +14,12 @@ class Meta:
             'moodle_base_url',
             'service_short_name',
             'category_id',
-            'username',
-            'password',
-            'token',
+            'encrypted_username',
+            'encrypted_password',
+            'encrypted_token',
         )
         fields = EnterpriseCustomerPluginConfigSerializer.Meta.fields + extra_fields
+
+    encrypted_password = serializers.CharField(required=False, allow_blank=False, read_only=False)
+    encrypted_username = serializers.CharField(required=False, allow_blank=False, read_only=False)
+    encrypted_token = serializers.CharField(required=False, allow_blank=False, read_only=False)
```

Comment:
This change will work even for customers for which the feature flag is not enabled. We can remove this change for now.

Reply:
@sameenfatima78 
i can remove these changes, but,
i thought the purpose of the feature flag was to not disturb the flow of api calls handled by the moodle client.
the encryption of these fields were already tested in the previous ticket,
only decryption was left which will be tested with the help of feature flag.

Reply:
We can move this change where we completely replace old fields with newer ones.

Reply:
@sameenfatima78 removed.